.pre-commit-config.yaml

```diff
@@ -37,12 +37,13 @@ repos:
   rev: v0.8.0
   hooks:
   - id: shellcheck
+    args: ["-x"]
 # Misc
 - repo: https://github.com/codespell-project/codespell
   rev: v2.2.2
   hooks:
   - id: codespell
-    args: ["--skip=*.pyc,tests/testdata/*,*.ipynb,*.csv"]
+    args: ["--skip=*.pyc,tests/testdata/*,*.ipynb,*.csv", "-L=reacher"]
```

Comment:
This does not feel like a great way to do it. Can we not have a config file with ignored words instead? Or does it make CI much more difficult?

Reply:
Yes I was kind of lazy here and felt that it's not worth to set up a new file just to ignore a single word.
It is totally possible to have a wordlist though. I will add it.